<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPgSourceSelectBase</class>
 <widget class="QDialog" name="QgsPgSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add PostGIS Table(s)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PostgreSQL Connections</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnNew">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QComboBox" name="cmbConnections"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="mTablesTreeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBuildQuery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Build query</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>271</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="mSearchOptionsButton">
     <property name="text">
      <string>Search options...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mSearchGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mSearchLabel">
        <property name="text">
         <string>Search:</string>
        </property>
        <property name="buddy">
         <cstring>mSearchTableEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mSearchModeLabel">
        <property name="text">
         <string>Search mode:</string>
        </property>
        <property name="buddy">
         <cstring>mSearchModeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="mSearchModeComboBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="mSearchColumnsLabel">
        <property name="text">
         <string>Search in columns:</string>
        </property>
        <property name="buddy">
         <cstring>mSearchColumnComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="mSearchColumnComboBox"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="mSearchTableEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>mTablesTreeView</tabstop>
  <tabstop>btnBuildQuery</tabstop>
  <tabstop>mSearchOptionsButton</tabstop>
  <tabstop>mSearchTableEdit</tabstop>
  <tabstop>mSearchColumnComboBox</tabstop>
  <tabstop>mSearchModeComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsPgSourceSelectBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
